examples/basic-setup-typescript/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "basic-setup-typescript",
+  "version": "1.0.0",
+  "description": "",
+  "main": "basic-setup-typescript.sketchplugin",
+  "manifest": "./manifest.json",
+  "scripts": {
+    "build": "skpm build",
+    "watch": "skpm build --watch",
+    "start": "npm run typescript & npm run render",
+    "render": "skpm build --watch --run",
+    "render:once": "skpm build --run",
+    "link-plugin": "skpm link",
+    "typescript": "./node_modules/.bin/tsc --watch",
+    "typescript:once": "./node_modules/.bin/tsc",
+    "typescript:clean": "rm -rf ./.ts-compiled"
+  },
+  "author": "Jon Gold <jon.gold@airbnb.com>",
+  "license": "MIT",
+  "devDependencies": {
+    "@types/chroma-js": "^1.3.3",
+    "@types/react": "^16.0.25",
+    "@types/react-sketchapp": "^0.12.0",
+    "skpm": "^0.10.2",
```

Comment:
probably should update to `@skpm/builder` (see #203 )

Reply:
Is that a one-line change?

Reply:
no there are some scripts to change as well: https://github.com/airbnb/react-sketchapp/pull/203/files#diff-3bcd169ea0fcdf0ddf5666461f84f003

Reply:
Alright 👍 I'll update once I have more time